SEED-platform/seed
# Added site-wide horizontal scrollbars if necessary

#### What's this PR do?
Modified page layout such that horizontal scrollbars are always visible, and the top title bar is now sticky regardless of scroll position
#### How should this be manually tested?
Go to pages with wide content (admin page, data quality rules, etc), shrink the browser width, and ensure that a horizontal scrollbar is always visible regardless of vertical scrollbar position.
#### What are the relevant tickets?
#1739
